Build broken with Qt < 4.7

d9fd60ee added a place holder to retroshare-gui/src/gui/chat/CreateLobbyDialog.ui, which was only added in Qt 4.7. CentOS is still using Qt 4.6 so the build is broken there.
To fix this the placeholder should be removed to the .ui file and added manually in the .cpp file with a check for the qt version.

Exact error:
[  680s] g++ -c -m64 -pipe -D_FILE_OFFSET_BITS=64 -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -D_REENTRANT -DRS_ENABLE_GXS -DRS_RELEASE_VERSION -DHAVE_XSS -DUBUNTU -DQT_NO_DEBUG -DQT_SCRIPT_LIB -DQT_XML_LIB -DQT_GUI_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I/usr/lib64/qt4/mkspecs/linux-g++-64 -I. -I/usr/include/QtUiTools -I/usr/include/QtCore -I/usr/include/QtNetwork -I/usr/include/QtGui -I/usr/include/QtXml -I/usr/include/QtScript -I/usr/include -Iretroshare-gui -I../../libretroshare/src -I../../libresapi/src -Itemp/moc -Itemp/ui -o temp/linux-g++-64/obj/ChatLobbyWidget.o gui/ChatLobbyWidget.cpp
[  681s] In file included from gui/chat/CreateLobbyDialog.h:6,
[  681s]                  from gui/ChatLobbyWidget.cpp:8:
[  681s] temp/ui/ui_CreateLobbyDialog.h: In member function 'void Ui_CreateLobbyDialog::retranslateUi(QDialog*)':
[  681s] temp/ui/ui_CreateLobbyDialog.h:249: error: 'class QLineEdit' has no member named 'setPlaceholderText'

--- a/retroshare-gui/src/gui/chat/CreateLobbyDialog.cpp
+++ b/retroshare-gui/src/gui/chat/CreateLobbyDialog.cpp
@@ -47,11 +47,10 @@ CreateLobbyDialog::CreateLobbyDialog(const std::set<RsPeerId>& peer_list, int pr
 
     ui->idChooser_CB->loadIds(IDCHOOSER_ID_REQUIRED, default_identity);
 
-//#if QT_VERSION >= 0x040700
-//	ui->lobbyName_LE->setPlaceholderText(tr("Put a sensible lobby name here")) ;
-//	ui->nickName_LE->setPlaceholderText(tr("Your nickname for this lobby (Change default name in options->chat)")) ;
-//#endif
-//	ui->nickName_LE->setText(QString::fromUtf8(default_nick.c_str())) ;
+#if QT_VERSION >= 0x040700
+	ui->lobbyName_LE->setPlaceholderText(tr("Put a sensible lobby name here"));
+	ui->lobbyTopic_LE->setPlaceholderText(tr("Set a descriptive topic here"));
+#endif
 
 	connect( ui->buttonBox, SIGNAL(accepted()), this, SLOT(createLobby()));
 	connect( ui->buttonBox, SIGNAL(rejected()), this, SLOT(close()));

--- a/retroshare-gui/src/gui/chat/CreateLobbyDialog.ui
+++ b/retroshare-gui/src/gui/chat/CreateLobbyDialog.ui
@@ -71,11 +71,7 @@
               </widget>
              </item>
              <item row="1" column="2">
-              <widget class="QLineEdit" name="lobbyTopic_LE">
-               <property name="placeholderText">
-                <string>Set a descriptive topic here</string>
-               </property>
-              </widget>
+              <widget class="QLineEdit" name="lobbyTopic_LE"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_4">

